README.md

```diff
@@ -15,3 +17,22 @@ To remove sourcerer from your machine:
 ```
 sudo sourcerer --uninstall
 ```
+
+Build
+=====
+
+To build and run this application, you'll need latest versions of Git, Gradle and JDK installed on your computer. From your command line:
+
```

Comment:
I would move this into Prerequisites section that will explain the user how to prepare his machine. Otherwise it looks good with me.